universal-login-sdk/src/core/services/WalletStorageService.ts

```diff
@@ -22,22 +21,20 @@ export class WalletStorageService implements WalletStorage {
   }
 
   async migrate() {
-    try {
-      const data = this.storageService.get(DEPRECATED_STORAGE_KEY);
-      if (data === null) return;
-      const {wallet} = cast(JSON.parse(data), asObject({
-        kind: asExactly('Deployed'),
-        wallet: asSerializedDeployedWallet,
-      }));
-      for (const network of ['mainnet', 'kovan', 'rinkeby', 'ropsten']) {
-        const providerService = new ProviderService(getProviderForNetwork(network));
-        if (await providerService.isContract(wallet.contractAddress)) {
-          this.storageService.remove(DEPRECATED_STORAGE_KEY);
-          this.storageService.set(`${DEPRECATED_STORAGE_KEY}-${network}`, data);
-          return;
+    for (const network of ['mainnet', 'kovan', 'rinkeby', 'ropsten']) {
+      try {
+        const data = this.storageService.get(`${DEPRECATED_STORAGE_KEY}-${network}`);
+        if (data === null) continue;
```

Comment:
Why we should continue? I believe we should return here

Reply:
Because we should migrate wallet for every network if we have such possibility. So if data is null we continue migrate other networks

Reply:
ok